### PR TITLE
Feature/df 2213  delete data on unregistering user

### PR DESCRIPTION
**Pull request by emina-str**
Logic is already present for sending an UserUnregisteredEvent.
Added rest endpoint for emitting for all unregistered user this event again.

OS: https://github.com/alv-ch/alv-onlineform-service/pull/112
JOBROOM: https://github.com/alv-ch/jobroom2/pull/907
CANDIDATE: https://github.com/alv-ch/candidate-service/pull/170
JOBAD: https://github.com/alv-ch/job-ad-service/pull/442

### CODE REVIEW

File: jobroom-gateway-app/src/main/java/ch/admin/seco/jobroom/service/impl/messaging/UserEventGateway.java [hunk 1]
```diff
@@ -35,10 +31,6 @@ void handleUserUnregisteredEvent(UserUnregisteredEvent userUnregisteredEvent) {
 
     @EventListener
     void handleUserChangedEvent(UserChangedEvent userChangedEvent) {
-        if (userChangedEvent.getPersonNumber() == null) {
-            LOGGER.info("UserEvent for User: {} did not contain a PersonNumber -> dropped it", userChangedEvent.getUserInfoId());
-            return;
-        }
         UserEventDto userChangedEventDto = UserEventAssembler.toUserEventDto(userChangedEvent);
```
Comment (hamidsoleymani) [inline jobroom-gateway-app/src/main/java/ch/admin/seco/jobroom/service/impl/messaging/UserEventGateway.java]: Why was this deleted?

Comment (emina-str) [reply jobroom-gateway-app/src/main/java/ch/admin/seco/jobroom/service/impl/messaging/UserEventGateway.java]: Because it was moved to where it actually belongs, in Onlineform-Service.
We should send all user events for all users.
